src/base.html

```diff
@@ -145,7 +145,7 @@
             href="https://fonts.googleapis.com/css?family={{
               font.text | replace(' ', '+') + ':300,400,400i,700%7C' +
               font.code | replace(' ', '+')
-            }}&display=fallback" />
+            }}&display=swap" />
```

Comment:
https://developers.google.com/web/updates/2016/02/font-display#swap
https://developers.google.com/web/updates/2016/02/font-display#fallback

Reply:
Okay since this body text, we'll want to keep fallback here